packaging/centos-6/build-image/inside/build-package

```diff
@@ -19,13 +19,11 @@ cd /tmp/build
 rm -rf build-dir
 mkdir build-dir
 cd build-dir
-cmake -DREALM_ENABLE_ENCRYPTION=1 \
-      -DREALM_NO_TESTS=1 \
-      -DCMAKE_INSTALL_PREFIX=/prod \
-      -DCMAKE_BUILD_TYPE=Release \
-      -GNinja ..
-ninja
-ninja install/strip
+cmake -D CMAKE_INSTALL_PREFIX=/prod \
+      -D CMAKE_BUILD_TYPE=Release \
+      -G Ninja ..
+ninja Core RealmConfig RealmImporter Realmd
+ninja install
```

Comment:
`ninja install` will build the prerequisite binaries before installing, I think building them explicitly on the previous line is redundant (this happens in a few places in this PR)

Reply:
Right... this was back from when I was testing another flag. Fixed :)